Source/ACE.Server/Factories/PlayerFactory.cs

```diff
@@ -379,14 +379,17 @@ private static WorldObject GetClothingObject(uint weenieClassId, uint palette, d
             return worldObject;
         }
 
-        private static void CreateIOU(Player player, uint missingWeenieId)
+        public static bool CreateIOU(Player player, uint missingWeenieId, bool withNetworking = false)
         {
             var book = (Book)WorldObjectFactory.CreateNewWorldObject("parchment");
 
             book.SetProperties("IOU", "An IOU for a missing database object.", "Sorry about that chief...", "ACEmulator", "prewritten");
             book.AddPage(player.Guid.Full, "ACEmulator", "prewritten", false, $"{missingWeenieId}\n\nSorry but the database does not have a weenie for weenieClassId #{missingWeenieId} so in lieu of that here is an IOU for that item.");
 
-            player.TryAddToInventory(book);
+            if (withNetworking == false)
```

Comment:
this should just be if (withNetworking), or if (!withNetworking) to check for false

if (withNetworking == false) is not a form commonly used